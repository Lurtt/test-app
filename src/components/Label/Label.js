import styled from 'styled-components'

export default styled.label`
  font-size: 0.7rem;
  font-weight: 700;
  text-transform: uppercase;
`
