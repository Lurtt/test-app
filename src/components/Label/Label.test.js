import React from 'react'
import renderer from 'react-test-renderer'

import Label from './'

describe('<Label />', () => {
  test('Should render', () => {
    const result = renderer.create(<Label>Hello world!</Label>).toJSON()
    expect(result).toMatchSnapshot()
  })
})
