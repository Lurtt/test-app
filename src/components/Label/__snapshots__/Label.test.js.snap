// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Label /> Should render 1`] = `
<label
  className="sc-bdVaJa jswEML"
>
  Hello world!
</label>
`;
